test_runner/performance/test_bulk_update.py

```diff
@@ -0,0 +1,55 @@
+import time
+
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+#
+# Benchmark estimating effect of prefetch on bulk update operations
+#
+def test_bulk_update(neon_env_builder: NeonEnvBuilder, zenbenchmark):
+
+    env = neon_env_builder.init_start()
+    n_records = 10000000
+
+    env.neon_cli.create_branch("test_bulk_update")
+    pg = env.postgres.create_start("test_bulk_update")
+    cur = pg.connect().cursor()
+    cur.execute("set statement_timeout=0")
+
+    cur.execute("create table t(x integer)")
+
+    with zenbenchmark.record_duration("insert-1"):
+        cur.execute(f"insert into t values (generate_series(1,{n_records}))")
+
+    cur.execute("vacuum t")
+    time.sleep(10)  # wait until pageserver catch-up
```

Comment:
`test_wal_acceptor.py` does this:
```
    lsn = Lsn(pg.safe_psql("SELECT pg_current_wal_flush_lsn()")[0][0])
    log.info(f"pg_current_wal_flush_lsn is {lsn}, waiting for it on pageserver")

    # wait for the pageserver to catch up
    wait_for_last_record_lsn(
        ps.http_client(auth_token=auth_token),
        tenant_id,
        timeline_id,
        lsn,
    )

```
That would be better than sleeping

Reply:
Fixed

Reply:
I have included them in #2879.
Do you wan to commit them independently from last written lsn changes?